src/reverse-proxy-handler.ts

```diff
@@ -115,22 +115,6 @@ export class ReverseProxyHandler implements ProxyHandler<ReverseProxyTarget> {
         const raw = env.getRawValue(sec);
         return raw as RawObject;
     }
-    has(shadowTarget: ReverseShadowTarget, key: PropertyKey): boolean {
-        return key in shadowTarget;
-    }
-    ownKeys(shadowTarget: ReverseShadowTarget): (string | symbol)[] {
-        return push(
-            getOwnPropertyNames(shadowTarget),
-            getOwnPropertySymbols(shadowTarget)
-        );
-    }
-    getOwnPropertyDescriptor(shadowTarget: ReverseShadowTarget, key: PropertyKey): PropertyDescriptor | undefined {
-        return getOwnPropertyDescriptor(shadowTarget, key);
-    }
-    getPrototypeOf(shadowTarget: ReverseShadowTarget): object {
-        // nothing to be done here since the shadowTarget must have the right proto chain
-        return getPrototypeOf(shadowTarget);
-    }
```

Comment:
Many of these traps can be removed ☝️ since they are just defining the default behavior of the trap and since we `null` the prototype of the handler we ensure the default behavior will be used anyways : )